doc/getting-started/roscube/roscube-gsg.rst

```diff
@@ -0,0 +1,720 @@
+   <https://releases.ubuntu.com/18.04/>`_ for example):
+
+#. Install necessary packages.
+
+   .. code-block:: bash
+
+     sudo apt install qemu-kvm libvirt-clients libvirt-daemon-system \
+       bridge-utils virt-manager ovmf sudo reboot
+
+Create User VM image
+====================
+
+.. note:: Please create User VM image on **native Linux kernel**,
+   not ACRN kernel, or you'll get an error message.
+
+#. Start virtual machine manager application.
+
+   .. code-block:: bash
+
+     sudo virt-manager
+
+#. Create a new virtual machine.
+
+   .. figure:: images/rqi-acrn-kvm-new-vm.png
+
+#. Select your ISO image path.
+
+   .. figure:: images/rqi-acrn-kvm-choose-iso.png
+
+#. Select CPU and RAM for the VM.  You can modify as high as you can to
+   accelerate the installation time.  The settings here are not related to
+   the resource of the User VM on ACRN, which can be decided later.
+
+   .. figure:: images/rqi-acrn-kvm-cpu-ram.png
+
+#. Select disk size you want. **Note that this can't be modified after creating image!**
+
+   .. figure:: images/rqi-acrn-kvm-storage.png
+
+#. Edit image name and select "Customize configuration before install".
+
+   .. figure:: images/rqi-acrn-kvm-name.png
+
+#. Select correct Firmware, apply it, and Begin Installation.
+
+   .. figure:: images/rqi-acrn-kvm-firmware.png
+
+#. Now you'll see the installation page of Ubuntu.
+   After installing Ubuntu, you can also install some necessary
+   packages, such as ssh, vim, and ROS 2.
+   We'll clone the image for real-time VM to save time.
+
+#. To install ROS 2, please refer to `Installing ROS 2 via Debian Packages
+   <https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/>`_
+
+#. Optional: Use ACRN kernel if you want to passthrough GPIO to User VM.
+
+   .. code-block:: bash
+
+     sudo apt install git build-essential bison flex libelf-dev libssl-dev liblz4-tool
+
+     # Clone code
+     git clone -b release_2.1 https://github.com/projectacrn/acrn-kernel
+     cd acrn-kernel
+
+     # Set up kernel config
+     cp kernel_config_uos .config
+     make olddefconfig
+     export ACRN_KERNEL_UOS=`make kernelversion`
+     export UOS="ROS2SystemUOS"
+     export BOOT_DEFAULT="${ACRN_KERNEL_UOS}-${UOS}"
+     sed -ri "/CONFIG_LOCALVERSION=/s/=.+/=\"-${UOS}\"/g" .config
+
+     # Build and install kernel and modules
+     make all
+     sudo make modules_install
+     sudo make install
+
+     # Update Grub
+     sudo sed -ri \
+       "/GRUB_DEFAULT/s/=.+/=\"Advanced options for Ubuntu>Ubuntu, with Linux ${BOOT_DEFAULT}\"/g" \
+       /etc/default/grub
+     sudo update-grub
+
+#. Poweroff the VM after complete.
+
+   .. code-block:: bash
+
+     sudo poweroff
+
+Run User VM
+===========
+
+Now back to the native machine to set up the environment for launching
+the User VM.
+
+#. Install dependency.
+   The origin version of iasl is too old for ACRN and should be upgraded.
+
+   .. code-block:: bash
+
+     sudo apt install iasl
```

Comment:
The "next few steps" only make and then copy the updated iasl binary.  Is there more to the iasl installation (such as man pages, etc.) that would be missed if the ``sudo apt install iasl`` isn't done first?  Or should there be a ``make install`` after the ``make iasl``?

Reply:
The `iasl` binary is all we need for ACRN. It's not really visible to the ACRN user so I don't feel we need the manpages. We do not do a `make install` because it does not install `iasl` under `/usr/sbin` and this path is hard-coded in our code. So even if we did a `make install`, we'd have to manually copy the binary again.